Gemfile

```diff
@@ -9,25 +9,31 @@ gem 'rails', '~> 7.0.2'
 gem 'bcrypt',            '~> 3.1' # Use ActiveModel has_secure_password
 gem 'bootsnap',          '~> 1.11', require: false # Reduces boot times through caching; required in config/boot.rb
 gem 'ddtrace',           '~> 0.54', require: 'ddtrace/auto_instrument' # Datadog's tracing client for Ruby
+gem 'faraday-http-cache', '~> 2.2' # middleware that respects HTTP cache by checking expiration of the stored responses.
```

Comment:
Indent off

Reply:
rubocop complains about no space after `,`. Should I move everything over to match this one?

Reply:
Yes.